test/MyTested.AspNetCore.Mvc.Controllers.ActionResults.Test/BuildersTests/ActionResultsTests/BadRequestTests/BadRequestTestBuilderTests.cs

```diff
@@ -507,5 +508,57 @@ public void AndAlsoShouldWorkCorrectly()
                         TestObjectFactory.GetOutputFormatter(), 
                         new CustomOutputFormatter()));
         }
+
+        [Fact]
+        public void PassingShouldCorrectlyRunItsAssertionFunction()
+        {
+            MyController<MvcController>
+                .Instance()
+                .Calling(c => c.FullHttpBadRequestAction())
+                .ShouldReturn()
+                .BadRequest(badRequest => badRequest
+                    .Passing(br => br.Formatters?.Count == 2));
+        }
+
+        [Fact]
+        public void PassingShouldThrowAnExceptionOnAnIncorrectAssertion()
+        {
+            Test.AssertException<InvocationResultAssertionException>(
+                () =>
+                {
+                    MyController<MvcController>
+                        .Instance()
+                        .Calling(c => c.FullHttpBadRequestAction())
+                        .ShouldReturn()
+                        .BadRequest(badRequest => badRequest
+                            .Passing(br => br.Formatters?.Count == 0));
+                }, 
+                $"When calling {nameof(MvcController.FullHttpBadRequestAction)} " +
```

Comment:
This line should be one to make it the same with all other tests. I know the line is long but I prefer to be consistent. Otherwise, we have to make all such line in all assemblies separated on multiple lines and it will take too much time. Additionally, I am not sure whether I use nameof in the messages because the project started when the operator was not available. If we introduce it now, it may become inconsistent.